R/common-find-variation.R

```diff
@@ -425,4 +425,498 @@ getPipedValue <- function(string) {
   } else {
     result
   }
+}
+
+
+#' @title
+#' Find variation across groups
+#' @description Plot a boxplot to compare the variation across the groups. 
+#' @param df A dataframe
+#' @param categoricalCols Vector of strings representing categorical column(s)
+#' @param measureColumn Vector of strings representing measure column(s)
+#' @param printPlot Optional, default is TRUE. FALSE: only shows the statistics  
+#' used to plot the boxplot. 
+#' @param printTable Optional, default is TRUE. FALSE: not to show the table of  
+#' mean/std and quartiles 
+#' @param dateCol Optional. A date(time) column to group by (done by month) 
+#' @return A boxplot to compare variation across groups.
+#' @export
+#' @references \url{http://healthcare.ai}
+#' @seealso \code{\link{healthcareai}} \code{\link{calculateCOV}} 
+#' \code{\link{createVarianceTallTable}}
+#' @examples
+#' df <- data.frame(Gender = factor(rbinom(100, 1, 0.45), labels = c("Male","Female")), 
```

Comment:
Can you add a comment in each example that says somethign like:
"The dataset has groups A, B, and C. Attributes are gender and date. Measure is length of stay."